packages/sample-app/src/app-minimal.css

```diff
@@ -0,0 +1,3 @@
+html, body, #app {
```

Comment:
@karelhala Sass (`.scss`) files require additional processing, while plain `.css` files do not.

Sample application's stylesheets are very simple; I wouldn't recommend introducing Sass unless we really need it.

The host application doesn't need to use Sass in order to consume the PatternFly base CSS stylesheet.

Reply:
It would simplify the process of loading PF stylesheets - you need to have additional config in webpack to include some CSS variables. Which could be missleading to a new developer.